F1WM/Controllers/DriversController.cs

```diff
@@ -0,0 +1,47 @@
+using F1WM.ApiModel;
+using F1WM.Services;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace F1WM.Controllers
+{
+	[Route("api/[controller]")]
+	public class DriversController : ControllerBase
+	{
+		private readonly IDriversService service;
+		private readonly ILoggingService logger;
+
+		[HttpGet]
+		[Produces("application/json", Type = typeof(Drivers))]
+		public async Task<IActionResult> GetDrivers([FromQuery(Name = "letter")] string letter)
```

Comment:
If it is a letter then it should be a char indeed. It will save us some checking and will make the intent clearer.